tests/CMakeLists.txt

```diff
@@ -230,6 +230,8 @@ if(WITH_GUI_TESTS)
     # CLI clip tests need X environment on Linux
     add_unit_test(NAME testcli SOURCES TestCli.cpp
             LIBS testsupport cli ${TEST_LIBRARIES})
+    add_unit_test(NAME testclimerge SOURCES TestCliMerge.cpp
+        LIBS testsupport cli  ${TEST_LIBRARIES})
```

Comment:
@wizzk42 why is the formatting different from above? Did you run `make format`?

Reply:
format does not touch CMakeLists files